cmake/finders/FindLibAJANTV2.cmake

```diff
@@ -56,11 +56,30 @@ if(PKG_CONFIG_FOUND)
 endif()
 
 find_path(
-  LibAJANTV2_INCLUDE_DIR
-  NAMES ajalibraries
+  _libajantv2_new_include_dir
+  NAMES libajantv2
   HINTS ${PC_LibAJANTV2_INCLUDE_DIRS}
   PATHS /usr/include /usr/local/include
-  DOC "LibAJANTV2 include directory")
+  DOC "LibAJANTV2 (new) include directory")
+if(${_libajantv2_new_include_dir} STREQUAL "_libajantv2_new_include_dir-NOTFOUND")
```

Comment:
Strings with the `-NOTFOUND` suffix are "falsy", so check for the positive case and use the negative case in the `else` branch.